genome/clust.py

```
@@ -0,0 +1,175 @@
+* @Description:
+"""
+# """
+
+import numpy as np
+import pandas as pd
+import scipy.cluster
+from scipy.spatial import distance as ssd
+
+from typing import Iterable, TypeVar
+
+import numpy as np
```

Ruff (0.11.9)

Redefinition of unused `np` from line 11

Remove definition: `np`

(F811)